linera-views-derive/src/lib.rs

```diff
@@ -81,13 +84,31 @@ fn context_and_constraints(
     (context, constraints)
 }
 
+/// Merges two optional [`WhereClause`]s, returning a new one with all the predicates from both of
+/// them.
+fn merge_where_clauses(
+    first: Option<WhereClause>,
+    second: Option<WhereClause>,
+) -> Option<WhereClause> {
```

Comment:
Ditto above, if we replace `Option<WhereClause>` with `WhereClause` this function becomes less noisy.

Reply:
Done.